.gitignore

```diff
@@ -2,6 +2,8 @@
 _junk/
 rclone
 rclone.exe
+/git-annex-remote-rclone-goyle
+/git-annex-remote-rclone-goyle.exe
```

Comment:
The line above begins with slash to disambiguate from the contrib/ subdirectory with the same name.

This one doesn't need to begin with a slash, I suppose.

Reply:
Leaving this as is, it doesn't seem super important.